tests/unit/ulyanov_et_al_2016/test_loss.py

```diff
@@ -68,10 +68,8 @@ def test_GramOperator(
         normalize,
     ) in configs:
         with subtests.test(impl_params=impl_params):
-            target_repr = pystiche.gram_matrix(
-                encoder(target_image), normalize=normalize
-            )
-            input_repr = pystiche.gram_matrix(encoder(input_image), normalize=normalize)
+            target_repr = pystiche.gram_matrix(encoder(target_image), normalize=True)
```

Comment:
Since `normalize=True` is fixed, there is no need to include it in the loop variables (L68).